pages/spec/features/refinery/pages_spec.rb

```diff
@@ -425,12 +425,12 @@ def standard_page_menu_items_exist?
         let(:nested_page_slug) { 'nested_page' }
 
         let!(:nested_page) do
-          Globalize.fallbacks = [:ru]
-          _page = Globalize.with_locale(:en) {
+          Mobility.new_fallbacks[:ru]
```

Comment:
Do you need this? I'm planning to deprecate `new_fallbacks`. I don't think this does anything here.

Reply:
If you want to just set the global fallbacks for this one spec, couldn't you set `I18n.fallbacks` instead?